src/frontend/src/lib/components/hero/about/AboutHow.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import AboutItem from '$lib/components/hero/about/AboutItem.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+
+	export let asMenuItem = false;
+	export let onClick: (() => void) | undefined = undefined;
+</script>
+
+<AboutItem
+	labelAsTitle={$i18n.about.how.text.title}
+	labelAsMenuItem={$i18n.about.how.text.sub_title}
+	openModal={modalStore.openAboutHow}
+	{asMenuItem}
+	{onClick}
```

Comment:
would it work by bubbling the `on:click`?

Reply:
you mean re-dispatching the event? like  `dispatch('click')` in AboutItem?

because i tried to call on:click from there and it didn't work:

`<li><AboutWhat asMenuItem on:click={() => (visible = false)} /></li>`

Reply:
in my first try it didn't work, but I found out that to bubble it i need to repeat the event

```
class={asMenuItem ? '' : 'text-center text-white text-lg font-bold px-4 whitespace-nowrap'}
on:click={handleClick}
on:click
```

Reply:
changed